teams: smoother members removal (fixes #4819)

deeppp15 opened the pull request:
## Description

- fixes #4819


## Screenshot


https://github.com/user-attachments/assets/5d126f04-73d4-4c11-87ab-8eff07faad0b

Okuro3499 reviewed (changes requested):
if you remove a team leader it is still not properly refreshed because if you exit the page and reopen you will see an accurate list
[Screen_recording_20241203_184929.webm](https://github.com/user-attachments/assets/8184d950-dca4-43ff-aa66-adee4b2b1498)

deeppp15 commented:
Hi @Okuro, I have fixed the make team leader, please check. Updated the binding configuration.
https://github.com/user-attachments/assets/84a95cf3-2686-4e6d-8c73-2cc24a870426
